compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -65,19 +70,30 @@ bool resolve_with_BroadcastTo(luci::CircleCustom *addv2)
   if (broadcastTo_idx == -1)
     return false;
 
-  auto input = loco::must_cast<const luci::CircleCustomOut *>(addv2->inputs(broadcastTo_idx));
-  auto broadcastTo = loco::must_cast<luci::CircleCustom *>(input->input());
-
   auto name = addv2->name();
   assert(name.length() > 0);
 
   auto add = addv2->graph()->nodes()->create<luci::CircleAdd>();
   add->fusedActivationFunction(luci::FusedActFunc::NONE);
   add->x(addv2->inputs(1 - broadcastTo_idx));
-  add->y(broadcastTo->inputs(0));
   add->name(name + "/Add");
-  luci::add_origin(
-    add, luci::composite_origin({luci::get_origin(broadcastTo), luci::get_origin(addv2)}));
+
+  auto input = dynamic_cast<const luci::CircleCustomOut *>(addv2->inputs(broadcastTo_idx));
+
+  if (input)
+  {
+    auto broadcastTo = loco::must_cast<luci::CircleCustom *>(input->input());
+    add->y(broadcastTo->inputs(0));
+    luci::add_origin(
+      add, luci::composite_origin({luci::get_origin(broadcastTo), luci::get_origin(addv2)}));
+  }
+  else
+  {
+    auto broadcastTo = dynamic_cast<luci::CircleBroadcastTo *>(addv2->inputs(broadcastTo_idx));
+    add->y(broadcastTo->input());
+    luci::add_origin(
+      add, luci::composite_origin({luci::get_origin(broadcastTo), luci::get_origin(addv2)}));
+  }
```

Comment:
This is `ResolveCustomOpAddPass` that processes `Net_BroadcastTo_AddV2` that was previously processed.
Previously, `BroadcastTo` came in as a custom, but with the addition of a new builtin `BroadcastTo`, it changed here.
I think there will be a better code, but I don't know how to change it. 🤔

Reply:
IMHO, leave existing pass `Custom(BroadcastTo)` and add new `BroadcastTo` that works like existing one.
@Seunghui98 , can you guess why do like this?

Reply:
@seanshpark 

Oh, I'm sorry.. I checked this late. 😅

> can you guess why do like this?

Well... I've been thinking about it, but I don't know why.
Could you give me a related file or hint to find the reason? :)

Reply:
1/ it's about change management. 
2/ you've introduce `if` which makes code more complicate
3/ most of all, this pass is about `CustomOp` + `Add` sequence(pattern) but your code is checking also `CircleBroadcastTo`

Reply:
@seanshpark 

Thank you for your explanation. :)
I thought there was a lot of code duplication, so I added `CircleBroadcastTo` in `ResolveCustomOpAddPass.cpp`. 
but considering the reason you explained, it would be better to add new `BroadcastTo` that works like existing one.

As mentioned in this issue, https://github.com/Samsung/ONE/issues/10784#issuecomment-1797862821
I will make pass files as follows.
```
- New pass that processes `BroadcastTo(built-in) - Custom(AddV2)` pattern
- New pass that changes custom op into built-in op
```